File: src/main/java/com/example/jobs/dto/DeveloperRegisterResponse.java
package com.example.jobs.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@JsonIgnoreProperties(ignoreUnknown = true)
public class DeveloperRegisterResponse {
  @JsonProperty("developer")
  private RegisteredDeveloperDto developer;

  @JsonProperty("message")
  private String message;
}

File: src/main/kotlin/com/example/jobs/dto/DeveloperRegisterResponse.kt
package com.example.jobs.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
class DeveloperRegisterResponse {
    @field:JsonProperty("developer")
    var developer: RegisteredDeveloperDto? = null

    @field:JsonProperty("message")
    var message: String? = null
}